src/components/PageTitle.vue

```diff
@@ -0,0 +1,66 @@
+<template>
+  <div class="spanner">
+    <div class="title-container">
+      <return-button />
+      <h3>{{title}}</h3>
+      <p>{{subtitle}}</p>
+    </div>
+  </div>
+</template>
+
+<script>
+import ReturnButton from './ReturnButton.vue';
+
+export default {
+  name: 'PageTitle',
+  components: {
+    ReturnButton,
+  },
+  props: {
+    title: {
```

Comment:
You can specify these are String props here too

Reply:
updated